scripts/customer_info.py

```diff
@@ -0,0 +1,40 @@
+import os
+import yaml
+
+# change this based on customer repo path
+repo_path = "/"
+
+# change path for targets as relevant, relative to repo
+bazel_target_command = "bazel cquery //java/..."
+
+# change path for actions as relevant, relative to repo
+bazel_action_command = "bazel aquery //java/..."
+
+
+def writeToFile(dict_file):
+    print(dict_file)
+    with open(r'scripts/customer_info.yaml', 'w') as file:
+        yaml.dump(dict_file, file)
+
+
+if __name__ == '__main__':
+    # dictionary with information
+    dict_file = {}
+
+    # move to customer project
+    os.chdir(repo_path)
```

Comment:
Lets set the `repo_path` from `sys.argv` please, so we pass it as first and only argument for now.